Spec/RealtimeClientConnection.swift

```diff
@@ -1859,9 +1870,11 @@ class RealtimeClientConnection: QuickSpec {
                     let options = AblyTests.commonAppSetup()
                     options.autoConnect = false
                     options.authCallback = { tokenParams, callback in
-                        callback(getTestTokenDetails(key: options.key, capability: tokenParams.capability, ttl: tokenParams.ttl), nil)
+                        delay(0) {
```

Comment:
These things could really use a comment. I have no idea why this delay. Our users also need to make sure they don't return right away from their authCallbacks? If so, we probably should put the delay in our code, not on theirs.